### PR TITLE
Changing groundzero location

### PR DESCRIPTION
**Pull request by ryangrundy7**
# Motivation and Context

Updated the groundzero script in the performance environment to use the groundzero from the ddl repo instead of the toolbox repo.

# What has changed

- Groundzero runs from ddl repo instead of toolbox

# Links
[Trello](https://trello.com/c/rnYxcW1Z)

### CODE REVIEW

**Comment by ryangrundy7** [comment]
[Performance environment needs to rebuild the DB using the groundzero script which matches the release it's testing (5)](https://trello.com/c/rnYxcW1Z/540-performance-environment-needs-to-rebuild-the-db-using-the-groundzero-script-which-matches-the-release-its-testing-5)

**Review by AdamHawtin — APPROVED** [review]

File: tasks/kubectl-run-groundzero-script.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+inputs:
+  - name: kubernetes-repo
+run:
+  path: sh
+  args:
+    - -exc
+    - |
+    - -exc
+    - |
```
Comment (AdamHawtin) [inline tasks/kubectl-run-groundzero-script.yml]: Duplicate lines

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

File: tasks/kubectl-run-groundzero-script.yml (hunk 1 above)
Comment (ryangrundy7) [reply tasks/kubectl-run-groundzero-script.yml]: Fixed

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**